test/test_utils/ColdResumeManager.cpp

```diff
@@ -138,9 +140,8 @@ ColdResumeManager::~ColdResumeManager() {
     f << jsonState;
     f.close();
   } catch (const std::exception& ex) {
-    LOG(ERROR) << "Persisting state to " << outputFile_ << " failed. "
-               << ex.what();
-    return;
+    throw std::runtime_error(folly::sformat(
```

Comment:
Should we give error, or LOG anything if the `output_file.empty()` branch is taken?

Reply:
Why not throw an error, saying that the filename is empty, as we already throw.